ql/src/test/results/clientpositive/autoColumnStats_4.q.out

```diff
@@ -128,16 +127,20 @@ STAGE PLANS:
             Statistics: Num rows: 10 Data size: 1728 Basic stats: COMPLETE Column stats: COMPLETE
             Group By Operator
               aggregations: compute_stats(a, 'hll'), compute_stats(b, 'hll')
-              minReductionHashAggr: 0.99
-              mode: hash
+              mode: complete
```

Comment:
Why is the GBy operator changing to complete mode here? That could potentially slow down the insert. Does it have anything to do with the size (numrows) of the insert?

Reply:
Enabling parallelism when inserting eliminates this change.